movie/src/components/Icons/DotIcon/DotIcon.jsx

```diff
@@ -3,9 +3,9 @@ import IconButton from "@mui/material/IconButton";
 import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
 import { Menu } from "@mui/material";
 import "./DotIcon.css";
-import DotPopup from "../../PopupCard/DotPopup/DotPopup";
+import DetailPopup from "../../PopupCard/DetailPopup/DetailPopup";
 
-export default function LongMenu() {
+const MoreHorizICon = () => {
```

Comment:
1. `....ICon` ---> `....Icon`
2. The component name should be the same as the file name